plugins/clnrest/clnrest.py

```diff
@@ -9,17 +9,20 @@
     import multiprocessing
     from gunicorn import glogging  # noqa: F401
     from gunicorn.workers import sync  # noqa: F401
-
+    from pyln.client import Plugin
     from pathlib import Path
-    from flask import Flask, request, Blueprint
-    from flask_restx import Api
+    from flask import Flask, request, Blueprint, make_response
+    from flask_restx import Api, Namespace, Resource
     from flask_cors import CORS
```

Comment:
ok this refactoring are development preferred but if the app does not grow also a single file is good

Reply:
The important thing here is to have all that is related to the plugin mechanism in one place:

- the definition of the object `plugin`,
- the startup options of the plugin,
- what do we do at the `init` request,
- which notification topics we subscribe to and
- how do we start the plugin.